bin/dipy_nlmeans

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/env python
```

Comment:
Not a specialist for that. I tried #!python and it works. Seem to be automatically changed to #!/usr/bin/python in the setup.